README.md

```diff
@@ -52,7 +53,9 @@ Question keys
     * `max_value` integer, sets the maximum value for number questions (default 100)
     * `integer_only` boolean, when `true` makes the number question accept only integer values instead of numbers
     * `format` sets JSON schema 'format' property for text questions (eg `email`, `uri`)
-* `number_of_items` sets the maximum number of items for list questions (and changes "Add another (N remaining)" list button, defaults to 10)
+* `number_of_items` sets the maximum number of items
+    * for list questions, the add button becomes "Add another (N remaining)", and the default is 10
+    * for checkboxes and checkbox tree questions, by default any number can be checked, but this sets the maximum
```

Comment:
Correct, I'll clarify the comment. (But the frameworks lib only controls the schema generation - so what we do here neither precludes client-side enforcement, nor could it make it happen. If someone did do the client-side thing, they might not expect to have to update a comment here - so I'll word carefully.)